site/specs-source/webRtc.json

```diff
@@ -499,7 +499,7 @@
                }
             ],
             "requestBody": {
-               "description": "Subscriptions the participant should be created with",
+               "description": "Optional set of subscriptions to set on the participant.\nCalling this endpoint with no/empty body will only add the participant to the session.\n\nNOTE: the request body for this endpoint is OPTIONAL and provided as a convenience to avoid calls to this and the Update Participant Subscriptions endpoint.",
```

Comment:
I would suggest:  ...participant.\nCalling this endpoint **_without a_** body will only add the...
because I think that an empty body ( {} ) will cause an error

Reply:
Just tested this again, and calling the add participant to session endpoint w/an empty body ( {} ) still successfully adds the participant to the session. Are we ok to keep the description text as it is now?